tests/regression/test_bcs.py

```diff
@@ -57,6 +57,7 @@ def test_assemble_bcs_wrong_fs(V, measure):
     W = FunctionSpace(V.mesh(), "CG", 2)
     with pytest.raises(RuntimeError):
         A = assemble(dot(u, v)*measure, bcs=[DirichletBC(W, 32, 1)])
+        print(A)  # F841
```

Comment:
Prefer no `print(A)` and squash the warning either by not assigning the result to a variable, or else `# noqa: F401` (I think).

Reply:
Fixed!